tests/FSharp.Compiler.ComponentTests/ErrorMessages/TailCallAttribute.fs

```diff
@@ -1481,7 +1464,10 @@ namespace N
         |> withLangVersionPreview
         |> compile
         |> shouldFail
-        |> withSingleDiagnostic (Error 842, Line 6, Col 11, Line 6, Col 19, "This attribute is not valid for use on this language element")
+        |> withDiagnostics [
```

Comment:
Why is there another error?

Reply:
I believe `warning` allow the analysis to continue as opposed to `error` which just stop the analysis. ?

Reply:
You are right, this was the case there.